<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrashBin</class>
 <widget class="QDialog" name="TrashBin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="DisplaytapsT">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>651</width>
     <height>541</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tPerson">
    <attribute name="title">
     <string>Person</string>
    </attribute>
    <widget class="QPushButton" name="pBPTBack">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>470</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="ListPersons">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>401</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>101</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Gender</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Age</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Birth</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Death</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Turing Year</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contribution</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pBPRestore">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>470</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBPEmpty">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>470</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Empty</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tComputer">
    <attribute name="title">
     <string>Computer</string>
    </attribute>
    <widget class="QTableWidget" name="ListComputers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>401</height>
      </rect>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>203</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Type</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>YearBuilt</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pBCTBack">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>470</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBCRestore">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>470</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBCEmpty">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>470</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Empty</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tLinks">
    <attribute name="title">
     <string>Links</string>
    </attribute>
    <widget class="QTableWidget" name="ListLinks">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>401</height>
      </rect>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>304</number>
     </attribute>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Computer</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pBLTBack">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>470</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBLRestore">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>470</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pBLEmpty">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>470</y>
       <width>80</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Empty</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
